deployability/modules/workflow_engine/__main__.py

```diff
@@ -1,10 +1,12 @@
+
```

Comment:
Remove this empty line

Reply:
Done in [ebce158](https://github.com/wazuh/wazuh-qa/pull/4960/commits/ebce158066ecff2dc0cb82cc7f902dc54b6d489d)